docs/source/using/managing-minishift.adoc

```diff
@@ -230,6 +230,24 @@ CAUTION: Driver specific options might overlap with values specified using Minis
 specific flags and environment variables. Examples are boot2docker URL, memory size, cpu count, etc.
 In this case driver specific environment variables will override Minishift specific settings.
 
+[[caching-openshift-images]]
+== Caching OpenShift images (experimental)
+
+To speed up provisioning of the OpenShift cluster and to minimize network traffic the core OpenShift images are cached on the host.
+Caching occurs transparently, in a background process, the first time you use the xref:../command-ref/minishift_start#minishift-ssh[`minishift start`] command.
+Once the images are cached under _$MINISHIFT_HOME/cache/images_, successive Minishift VM creations will use these cached images.
+
+Each time an image exporting background process runs a log file is generated under _$MINISHIFT_HOME/logs_ which can be used to verify the progress of the export.
+
+You can skip the caching of the OpenShift images via setting _skip-image-caching_ to _true_ (see also xref:./managing-minishift#setting-persistent-configuration-values[setting persistent configuration values]):
+
+----
+$ minishift config set skip-image-caching true
+----
+
+NOTE: Image caching is considered an experimental and its semantics and API is subject to change. The aim is to allow caching of arbitrary images, as well as using a better format for storing the images on the host. You can track the progress on this feature on the
```

Comment:
After giving some more thoughts I think we should remove `considered an` i.e. Image caching is experimental and its semantics and API is subject to change.